src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details.html.twig

```diff
@@ -111,14 +111,16 @@
   </div>
 </div>
 
-<div class="mt-2" id="order_hook_tabs">
-  <ul class="nav nav nav-tabs" role="tablist">
-    {# Rendering of hook displayAdminOrderTabLink, we expect tab links #}
-    {{ renderhook('displayAdminOrderTabLink', {'id_order': orderForViewing.id}) }}
-  </ul>
+{% if hookcount('displayAdminOrderTabLink') > 0 %}
```

Comment:
Although both hooks are supposed to be related and have the same count, maybe it's better to separate them with each one checking its own count This way if a module forgets to add the appropriate link it will at least display the content, which is less confusing for debugging

Reply:
Added an or condition